content/techniques/task-scheduling.md

```diff
@@ -53,13 +53,32 @@ export class TasksService {
 
 In this example, the `handleCron()` method will be called each time the current second is `45`. In other words, the method will be run once per minute, at the 45 second mark.
 
+Some of you might consider defining a simple cron pattern with a plain string is a cumbersome task to be done. You can use the `CronExpression` enum instead, which provides commonly used cron patterns, as follows:
+
+```typescript
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+
+@Injectable()
+export class TasksService {
+  private readonly logger = new Logger(TasksService.name);
+
+  @Cron(CronExpression.EVERY_45_SECONDS)
+  handleCron() {
+    this.logger.debug('Called every 45 seconds');
+  }
+}
+```
+
+In this example, the `handleCron()` method will be called every `45` seconds.
+
 The `@Cron()` decorator supports all standard [cron patterns](http://crontab.org/):
 
 - Asterisk (e.g. `*`)
 - Ranges (e.g. `1-3,5`)
 - Steps (e.g. `*/2`)
 
-In the example above, we passed `45 * * * * *` to the decorator. The following key shows how each position in the cron pattern string is interpreted:
+In the first example above, we passed `45 * * * * *` to the decorator. The following key shows how each position in the cron pattern string is interpreted:
```

Comment:
I think if we make the change above (moving to the bottom), this change can be reverted.